res/TensorFlowPythonExamples/examples/PadV2/__init__.py

```diff
@@ -0,0 +1,7 @@
+import tensorflow as tf
+import numpy as np
+
+input_ = np.ones(shape=[1, 1, 1, 1], dtype=np.float32)
+paddings_ = np.array([[1, 1], [2, 2], [3, 3], [4, 4]], dtype=np.int32)
+constant_values_ = 1
+op_ = tf.raw_ops.PadV2(input=input_, paddings=paddings_, constant_values=constant_values_)
```

Comment:
With TF 1.13.1, I got this error
```
AttributeError: module 'tensorflow' has no attribute 'raw_ops'
```

Reply:
This is expected behavior for TF less 2.0. Do you want more readable message for test?

Reply:
Our default TF is 1.13.1. If fails in 1.13.1 we cannot use this example.

Reply:
I'm sorry. I cannot find `PadV2` for `TF 1.13.2`. `PadV2` - a new operator which not available on `TF 1.13.2`. I don't have any ideas for fix it.

Reply:
Please add a comment at the top like
```
# NOTICE Run this script in TensorFlow 2.x as PadV2 is not available in 1.13.2
```

Reply:
@seanshpark 
I checked this operator in tf 1.13 again and found solution, now it works.